x-pack/plugin/profiling/src/main/java/org/elasticsearch/xpack/profiling/GetStackTracesRequest.java

```diff
@@ -98,7 +98,7 @@ public void writeTo(StreamOutput out) {
         TransportAction.localOnly();
     }
 
-    public Integer getSampleSize() {
+    public int getSampleSize() {
         return sampleSize != null ? sampleSize : DEFAULT_SAMPLE_SIZE;
```

Comment:
IMO, we can make `sampleSize` an `int` and move the null check into the constructor.